db/migrate/20191101150844_add_ontology_tag_system.rb

```diff
@@ -0,0 +1,29 @@
+class AddOntologyTagSystem < ActiveRecord::Migration[5.2]
+  def change
+    ## Add column to store ontology tag on organization
+    add_column :organizations, :ontology_tag, :string
+
+    ## Populate new ontology tag permissible values
+    unless PermissibleValue.where(category: 'ontology_tag').exists?
```

Comment:
Instead of creating the PermissibleValues here, can you add a source CSV file and run the import task?